api/k8s/v1/k8s.proto

```diff
@@ -545,6 +561,41 @@ message ListConfigMapsResponse {
   repeated ConfigMap config_maps = 1;
 }
 
+message DescribeConfigMapRequest {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.ConfigMap",
+    pattern : "{cluster}/{namespace}/{name}"
+  };
+
+  string clientset = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  string cluster = 2 [ (validate.rules).string = {min_bytes : 1} ];
+  string namespace = 3 [ (validate.rules).string = {min_bytes : 1} ];
+  string name = 4 [ (validate.rules).string = {min_bytes : 1} ];
+
+  map<string, string> labels = 5 [ (validate.rules).map.keys.string = {min_bytes : 1} ];
```

Comment:
Should we add annotations too?

Reply:
sure! added in https://github.com/lyft/clutch/pull/930/commits/147f0d0b6cd71926d777622693c9b8e90e5b7349